cli/waiter/cli.py

```diff
@@ -3,7 +3,7 @@
 from urllib.parse import urlparse
 
 from waiter import configuration, http_util, metrics, version
-from waiter.subcommands import create, delete, ssl, init, kill, maintenance, ping, show, ssh, tokens, update
+from waiter.subcommands import create, delete, init, kill, maintenance, ping, show, ssh, ssl, tokens, update
```

Comment:
We should separate formatting changes from functional changes

Reply:
Happy to move this to a separate PR if you prefer, it is a leftover from my attempt at cleaning the capstone branch and this showed up as a merge conflict.

Reply:
Moved the import change to a different PR: #1697